packages/mediapipe-task-text/lib/src/tasks/text_classification/text_classification.dart

```diff
@@ -0,0 +1,6 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+export 'text_classifier.dart';
+export 'containers/containers.dart';
```

Comment:
Does this need to export containers if it is already exported by `packages/mediapipe-task-text/lib/mediapipe_text.dart`?

Reply:
Relevant code has been refactored.